Makefile

```diff
@@ -130,3 +146,10 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+code-generator:
+  bash $(GOPATH)/src/k8s.io/code-generator@v0.21.5/generate-groups.sh \
```

Comment:
This needs to be a tab indent. Code generator also needs to be added to the `make generate` cadence.

Reply:
Added code-generator download task
https://github.com/keikoproj/addon-manager/pull/111/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R151